packages/pds-ember/addon/components/pds/avatar/index.hbs

```diff
@@ -1,3 +1,8 @@
+{{!--
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+--}}
+
```

Comment:
@alex-ju I just noticed that here the tool adds an empty line after the comment, while in our HDS repo it doesn't. Do you think this may be something potentially problematic on the consumers side? (eg. in Cloud UI)?

Reply:
I've not found any instances where the whitespace needs to be controlled (looked for things like `{{~` so new lines should be safe. It was me being overcautious in HDS, that's why there are no spaces there 😁